lib/sequent/support/database.rb

```diff
@@ -0,0 +1,63 @@
+module Sequent
+  module Support
+    # Offers support operations for a postgres database.
+    #
+    # Class methods do establish their own database connections (and therefore
+    # take in a database configuration). Instance methods assume that a database
+    # connection yet is established.
+    class Database
+      attr_reader :db_config
+
+      def self.create!(db_config)
+        ActiveRecord::Base.establish_connection(db_config.merge('database' => 'postgres'))
+        ActiveRecord::Base.connection.create_database(db_config['database'])
+      end
+
+      def self.drop!(db_config)
+        ActiveRecord::Base.establish_connection(db_config.merge('database' => 'postgres'))
+        ActiveRecord::Base.connection.drop_database(db_config['database'])
+      end
+
+      def self.establish_connections(db_config, view_schema_name)
+        Sequent::Support::BaseViewModel.establish_connection(
```

Comment:
This won't work imho since in this setup the BaseViewModel and event store classes have different connection pools. 
Then we can no longer commit and update to the view model and event store in the same transaction, because they use different connections.
This is fine we we **only** want to support eventual consistency, but that is not the case.